### PR TITLE
Separation power for Pions and Electrons

### PR DESCRIPTION
**Pull request by HorstMa**
This PR adds the separation power that can be trended for pions and electrons at the MIP point.
This is related to the O2 PR https://github.com/AliceO2Group/AliceO2/pull/12988

### CODE REVIEW

File: Modules/TPC/src/PID.cxx [hunk 1]
```diff
@@ -71,7 +74,10 @@ void PID::initialize(o2::framework::InitContext& /*ctx*/)
       getObjectsManager()->startPublishing(canv.get());
     }
   }
-}
+  cSeparationPower = mQCPID.getSeparationPower();
+  getObjectsManager()->startPublishing(cSeparationPower);
```
Comment (wiechula) [inline Modules/TPC/src/PID.cxx:78]:
```suggestion
  mSeparationPower = mQCPID.getSeparationPower();
  getObjectsManager()->startPublishing(mSeparationPower);
```

File: Modules/TPC/src/PID.cxx [hunk 2]
```diff
@@ -97,7 +103,45 @@ void PID::monitorData(o2::framework::ProcessingContext& ctx)
...
+        // Set initial parameters for the fit
+        fitFunc->SetParameter(0, 3000); // Amplitude of the first Gaussian
+        fitFunc->SetParameter(1, 50);   // Mean of the first Gaussian
+        fitFunc->SetParLimits(1, 45, 55);
+        fitFunc->SetParameter(2, 2);   // Sigma of the first Gaussian
+        fitFunc->SetParameter(3, 100); // Amplitude of the second Gaussian
+        fitFunc->SetParameter(4, 75);  // Mean of the second Gaussian
+        fitFunc->SetParLimits(4, 60, 90);
+        fitFunc->SetParameter(5, 10); // Sigma of the second Gaussian
+
```
Comment (wiechula) [inline Modules/TPC/src/PID.cxx:124]: Why is this inside the loop? The same fit function is used. So either:
- put it on the stack, or
- move it outside the and make it a `unique_ptr`.

File: Modules/TPC/src/PID.cxx [hunk 3]
```diff
@@ -97,7 +103,45 @@ void PID::monitorData(o2::framework::ProcessingContext& ctx)
...
+        mTrendingParameters.emplace_back(fitFunc->GetParameter(5)); // sigma electro
+      }
+    }
+  }
+
   ILOG(Debug, Devel) << "endOfCycle" << ENDM;
+
+  TPaveText* pSeparationPower = new TPaveText(.05, .05, .95, .95);
+  pSeparationPower->AddText(fmt::format("Mean Pi: {:.3}", mTrendingParameters[0]).c_str());
+  pSeparationPower->AddText(fmt::format("Mean El: {:.3}", mTrendingParameters[2]).c_str());
```
Comment (wiechula) [inline Modules/TPC/src/PID.cxx:141]: Add sigmas.

**Review by wiechula — CHANGES_REQUESTED** [review]

**Comment by HorstMa** [comment]
@knopers8 They are related to the O2 PR which has not yet been merged https://github.com/AliceO2Group/AliceO2/pull/12988

**Review by wiechula — APPROVED** [review]

**Pull request merged**